app/views/ranks/index.html.erb

```diff
@@ -0,0 +1,49 @@
+<p id="notice"><%= notice %></p>
+
+<h1>Listing Ranks</h1>
+
+<table>
+	<thead>
+		<tr>
+			<th>UserID</th>
+			<th>Score</th>
+			<th colspan="3"></th>
+		</tr>
+	</thead>
+
+	<div id="ranking">
+		<% counter = 1%>
+		<% rankedPosition = RanksController.rankList(2, 1)%>
+		<% rankedPosition.keys.each do |rank| %>
```

Comment:
Das ist sehr un-Ruby-ish. DIese beiden Zeilen kann man auch so schreiben:

    rankedPosition.each do |rank, pos|

Bitte in der Ruby-Doku nachlesen, wie genau und wieso das geht. ;)